egui_glow/src/painter.rs

```diff
@@ -32,6 +32,15 @@ impl Default for TextureFilter {
     }
 }
 
+impl From<egui::epaint::textures::TextureFilter> for TextureFilter {
```

Comment:
```suggestion
impl From<egui::epaint::textures::TextureFilter> for glow::TextureFilter {
```

for extra clarity

Reply:
This would be either `crate::painter::TextureFilter` or `self::TextureFilter`, because the struct is defined above.

Do you want it to be either of those?

Reply:
Oh, I forgot about `egui_glow::TextureFilter`. Let's just remove that and replace it with `pub type TextureFilter = gui::epaint::textures::TextureFilter;`